src/deploy/table_legal_term.sql

```diff
@@ -0,0 +1,69 @@
+-- Deploy maevsi:table_legal_term to pg
+-- requires: schema_public
+-- requires: role_account
+-- requires: role_anonymous
+
+BEGIN;
+
+CREATE TABLE maevsi.legal_term (
+  id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+  created_at  TIMESTAMPTZ NOT NULL DEFAULT NOW(),
```

Comment:
Other tables call this column just "created". We should decide whether to use "created" or "created_at" uniformly.

Reply:
It seems to me that `created_at` is the general convention, so I'd start using this here. I agree that we should eventually migrate all usages to `created_at`, which would be a breaking change though as the frontend queries would need to be updated. I think we can resolve this here, yes?

Reply:
Yes, let's call it resolved. But let'a also keep in mind that we would like to change the (few) occurences of `created` into `created_at` as soon as there is an opportunity for this.